src/views/partials/example.njk

```diff
@@ -12,6 +12,18 @@
 {% generateHTML examplePath %}
 {% endset %}
 
+{% set componentLabel = params.item + "-" + params.example %}
```

Comment:
I find nunjucks really clunky for this kind of stuff. Not saying you need to do it here but if you'd prefer to use JS to write view logic then you can [pair a template with a data file](https://www.11ty.dev/docs/data-template-dir/) and then expose values to the template.

Reply:
Happy to leave as is for now, but I might explore this again if we need to do more from the example component down the line